lib/src/json_schema/json_schema.dart

```diff
@@ -2023,6 +2031,14 @@ class JsonSchema {
   _setRequiredV6(dynamic value) =>
       _requiredProperties = (TypeValidators.list('required', value))?.map((value) => value as String)?.toList();
 
+  _setUnevaluatedItems(dynamic value) {
+    if (value is Map || value is bool && schemaVersion >= SchemaVersion.draft6) {
```

Comment:
Do order of operations happen here as we would hope? Otherwise I could see this code accidentally setting the property in draft 4.

Reply:
This code is also kind of an older pattern we should consider updating in the future.

Reply:
This code is also kind of an older pattern we should consider updating in the future.